tests/test_tasks.yaml

```diff
@@ -85,6 +85,8 @@
     ARA_SETTINGS: "{{ ara_api_settings }}"
     ARA_API_CLIENT: "{{ ara_api_client | default('offline') }}"
     ARA_API_SERVER: "{{ ara_api_server | default('http://127.0.0.1:8000') }}"
+    ARA_API_USERNAME: "{{ ara_api_username | default('') }}"
+    ARA_API_PASSWORD: "{{ ara_api_password | default('') }}"
```

Comment:
All jobs are failing with the callback returning HTTP 401 unauthorized errors, maybe these are being interpreted and used with empty values. Will check.